init.lua

```diff
@@ -160,6 +160,7 @@ function frame.register(name)
 		paramtype2 = "wallmounted",
 		sunlight_propagates = true,
 		walkable = false,
+		use_texture_alpha = "clip",
```

Comment:
Please use something like
```Lua
local ALPHA_CLIP = minetest.features.use_texture_alpha_string_modes and "clip" or true
```
to keep compatibility with clients < 5.4.0 because this mod does not state any minimal Minetest version.